### PR TITLE
refactor contact endpoint

### PR DESCRIPTION
**Pull request by Qalifah**

### CODE REVIEW

File: pkg/handler/response.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package handler
+
+type Response struct {
```
Comment (rubenhorn) [inline pkg/handler/response.go:3]: Shouldn't this be in the folder /internal/models?

File: internal/contact/email.go [hunk 2]
```diff
@@ -32,17 +32,17 @@ func NewContactService(emailRepository EmailRepository, configManager ConfigMana
 	return &contact{emailRepository: emailRepository, configManager: configManager, stage: stage}
 }
 
-func (e *contact) SendEmail(ctx context.Context, data models.EmailContactRequest) error {
+func (e *contact) SendEmail(ctx context.Context, req models.EmailContactRequest) error {
```
Comment (rubenhorn) [inline internal/contact/email.go:35]: I'm not sure if I agree with this renaming, but I suppose that's just a matter of preference.

File: pkg/handler/aws/emailContact/main.go [hunk 3]
```diff
@@ -21,8 +23,8 @@ func handler(ctx context.Context, req events.APIGatewayV2HTTPRequest) (events.AP
```
Comment (rubenhorn) [inline pkg/handler/aws/emailContact/main.go:24]: It is not part of this PR, but could you add a comment here: `// TODO implement POW verification using nonce (see https://github.com/CollActionteam/collaction_backend/issues/58)`

**Review by rubenhorn — COMMENTED** [review]
This was not part of this PR, but can you also (re-)implement the [verification of the app version field](https://github.com/CollActionteam/collaction_backend/blob/55d2f05a62ff36378857e8eb006bda6aef17c3d3/internal/models/contact.go#L7)?

File: pkg/handler/response.go (hunk 1 above)
Comment (Qalifah) [reply pkg/handler/response.go:3]: I thought about that, but my latter thought was that, since it's being used by the handlers alone. it might be okay if it resides in the handler pkg. or what do you think?

File: internal/contact/email.go (hunk 2 above)
Comment (Qalifah) [reply internal/contact/email.go:35]: okay

File: pkg/handler/aws/emailContact/main.go (hunk 3 above)
Comment (Qalifah) [reply pkg/handler/aws/emailContact/main.go:24]: okay

File: pkg/handler/response.go (hunk 1 above)
Comment (rubenhorn) [reply pkg/handler/response.go:3]: That is actually a good point.  
I might be shared between other handler implementations (since they also have to conform to the API), but I think this is in this case down to personal preference.

You don't need to change it.

Comment (Qalifah) [reply pkg/handler/response.go:3]: okay.

**Comment by Qalifah** [comment]
> This was not part of this PR, but can you also (re-)implement the [verification of the app version field](https://github.com/CollActionteam/collaction_backend/blob/55d2f05a62ff36378857e8eb006bda6aef17c3d3/internal/models/contact.go#L7)?

okay. I will check it out.

**Review by rubenhorn — APPROVED** [review]

**Comment by rubenhorn** [comment]
⚠ Please notify the frontend team about the changed endpoint in the API when merging.
